tensorflow/core/nccl/nccl_manager_test.cc

```diff
@@ -663,8 +663,8 @@ TYPED_TEST(NcclManagerTest, MultiNodeBroadcast) {
   this->RunMultiNodeBroadcastTest(num_nodes, num_ranks_per_node,
                                   /*src_node=*/0, /*src_local_rank=*/0,
                                   /*in_place=*/true);
-#endif
 }
+#endif
```

Comment:
How was this even compiling before?

Reply:
This test has not been compiling (in ROCm mode) since this commit broke it : 
https://github.com/tensorflow/tensorflow/commit/d65a5f1bdf5dc536efe9cfcdd64acc2d6273b6e2

Reply:
@cheshire, separated out this commit into PR #36017